pypi_formula_mappings.json

```diff
@@ -219,6 +219,10 @@
   "http-prompt": {
     "exclude_packages": ["six", "PyYAML"]
   },
+  "huggingface-cli": {
+    "package_name": "huggingface-hub",
```

Comment:
Any reason why we're not calling this `huggingface-hub` (or `huggingface_hub`) so that you get the same thing from a `brew install` and `pip install`?

Reply:
Because the installed script is named `huggingface-cli`, and I anticipate that Homebrew users will install this formula because they intend to use that script. In that sense, the name of the Python module is not really relevant.